interactive/search_app.py

```diff
@@ -0,0 +1,72 @@
+import streamlit as st
+
+from loguru import logger
+
+from interactive.st_utils import StatesManager, WidgetManager
+from interactive.app_utils import ModuleManager
+
+
+def main_streamlit(module_manager: ModuleManager):
```

Comment:
Why it this separate function? Can't you just create single class for managing all frontend (composition) and add something like `run` function?

Reply:
Done